### DISCUSSION TITLE
Champ additionnel visible malgré case non cochée - 3.4.3c

### DISCUSSION THREAD

**Discussion by acdct** [JeromeDevome/GRR#305] [Q&A] [answered]
Bonjour,
Nous avons installé la version 3.4.3c et avons ajouté un champ additionnel.
"Afficher le contenu dans les vues journées, semaine et mois" est bien décochée, mais lorsque nous allons sur le calendrier, nous voyons quand même l'information.
![Screenshot 2023-10-16 at 15-09-06 Rectorat de Paris](https://github.com/JeromeDevome/GRR/assets/125655169/98741af8-04d0-460e-a6ae-8c0003add88c)
![Screenshot 2023-10-16 at 15-08-21 Rectorat de Paris](https://github.com/JeromeDevome/GRR/assets/125655169/cc6c4f21-c5dc-4516-8f5b-8860b6a69dc3)

Merci pour votre aide.

Cordialement

**Comment by ynaessens** [comment, marked as answer]
Bonjour,
si vous avez un compte administrateur, les champs additionnels sont, en principe, toujours affichés.
S'ils le sont aussi avec un compte visiteur ou utilisateur, c'est une anomalie. Pouvez-vous confirmer ?
Par mesure de sécurité, il serait souhaitable d'installer la version 3.5.1 dans sa dernière version.
Cordialement,
YN

**Comment by acdct** [comment]
Bonjour,
En effet c'était à cause du compte administrateur. Avec un compte usager l'information ne s'affiche pas. Merci beaucoup.
Nous sommes pour le moment limité en montée de version à cause du serveur.
Merci beaucoup.
Cordialement

**Comment by ynaessens** [comment]
la version 3.5.1 a été testée avec php 5.6 et semble fonctionnelle dans ce contexte.
Là aussi, par sécurité, une version plus récente de php ne serait pas un luxe.
Merci pour votre retour,
cordialement,
YN

**Reply by acdct** [reply]
Je suis tout à fait d'accord avec vous...J'en ai fait la demande mais j'attends encore...
Nous sommes bloqués à cette version à cause du utf8mb4_unicode_ci non supporté.

**Comment by ynaessens** [comment]
Ah oui, soit c'est au niveau de MySQL, soit le module php mbstring qui est manquant...
Dans l'attente d'une mise à niveau, n'hésitez pas à faire remonter d'éventuels dysfonctionnements de la v3.4.3c
Cordialement,
YN

**Reply by acdct** [reply]
Il n'y a pas de souci. Merci !
